scheduler/scheduler-server/src/main/java/org/ow2/proactive/utils/JettyStarter.java

```diff
@@ -146,25 +257,29 @@ private static void startServer(Server server, String schedulerHost, int restPor
                 if (server.isStarted()) {
                     printDeployedApplications(server, schedulerHost, restPort, httpProtocol);
                 } else {
-                    logger.error("Failed to start web modules (REST API, portals)");
-                    System.exit(-1);
+                    logger.error("Failed to start web applications");
+                    System.exit(1);
                 }
             }
         } catch (BindException bindException) {
-            logger.error("Failed to start web modules (REST API, portals), port " + restPort +
+            logger.error("Failed to start web applications. Port " + restPort +
                     " is already used", bindException);
-            System.exit(-1);
+            System.exit(2);
         } catch (Exception e) {
-            logger.error("Failed to start web modules (REST API, portals)", e);
-            System.exit(-1);
+            logger.error("Failed to start web applications", e);
+            System.exit(3);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1147)